tests/builder-onboarding-telemetry.unit.test.ts

```
@@ -0,0 +1,143 @@
+import { afterEach, beforeEach, describe, expect, it, vi } from 'vitest'
+
+const sendEventMock = vi.hoisted(() => vi.fn())
+
+vi.mock('../cli/src/utils.ts', () => ({
+  sendEvent: sendEventMock,
+}))
+
+import { trackBuilderOnboardingStep } from '../cli/src/build/onboarding/telemetry.ts'
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)